JamesRexMiller4/BYOB
# Database

### What's this PR do?
This PR will create and run three migrations: the initial, dropping the followers/following columns of the Users table and increasing the default string length of the content column in the tweets table

In addition, this pr creates a seed file that contains the logic for iterating over the data.js file and goes about inserting each user data into the Users table, then inserting all that user's tweets data into the Tweets table, linking them to the corresponding tweets table. 

There may also be some restructuring of the directories of the project, putting the db folder and its seed and migrations directories inside of the platform directory

### Where should the reviewer start?
A reviewer should start by reviewing the data in the data.js file and once they are familiar with the structure of the data, go to the migrations folder and review each migration and what it does. Once they feel comfortable with how the tables will be structured in the database, they should go to the seeds folder and review the tweets file. There they should be able to see how the data from the data.js file is being iterated over, taking each user and inserting their data into the corresponding columns. Then taking that user's tweets and inserting them into the Tweets tables and linking them via the user_id. 

### How should this be manually tested?
No tests are currently available, and the database is being stored locally on my machine. So sorry, you will have to track me down and use my computer to verify the database is storing the data correctly.

### Any background context you want to provide?
It took 6 times to successfully seed the data, and there are 30 users, so the unique id's start at 181.
